algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -290,13 +290,13 @@ public Integer getAroundPrecision() {
     }
 
     private static final String KEY_AROUND_RADIUS = "aroundRadius";
-    public static final Integer RADIUS_ALL = Integer.MAX_VALUE;
+    public static final int RADIUS_ALL = Integer.MAX_VALUE;
 
     /**
      * Change the radius for around latitude/longitude queries.
      * @param radius the radius to set, or Query.RADIUS_ALL to disable stopping at a specific radius.
      */
-    public @NonNull Query setAroundRadius(Integer radius) {
+    public @NonNull Query setAroundRadius(int radius) {
```

Comment:
Sorry to disagree again, but the `int` was for the constant. I think the accessors need to be typed `Integer` because the value may be `null`.